gobase/intermediate.go

```diff
@@ -59,34 +59,35 @@ func CollectionsAvailableArr(address *string) ([]string, bool) {
 }
 
 //SaveArr ...
-func SaveArr(path *string, dataArr []string) bool {
-
-	exists := collectionStatus(*path)
-	if exists == false {
-		fmt.Println("No Collection existing at the specified datapath. Creating one ...")
-		createCollection(*path)
-	}
-	var address = "warehouse/" + *path + ".data"
-	file, err := os.OpenFile(address, os.O_WRONLY, 0600)
-	file.Seek(0, 0)
-	file.Truncate(0)
-	if err != nil {
-		panic(err)
-	}
-	defer file.Close()
-	var dataString = "["
-	dataString += strings.Join(dataArr, ",")
-	dataString += "]"
-	_, err = file.WriteString(dataString)
-	if err != nil {
-		fmt.Println("Error occured while writing the following data:")
-		fmt.Println("\n" + dataString)
-		fmt.Println("Address: warehouse/"+address)
-		fmt.Println(err)
-		return false
-	}
-	return true
-
+func SaveArr(path *string, dataArr *[]string) bool {
+
+	// exists := collectionStatus(*path)
+	// if exists == false {
```

Comment:
@muskankhedia  , why so many comments?

Reply:
You check it once, if the code will be fine, I will remove the previous code. Commented only for back-up purpose.

Reply:
Okay